### PR TITLE
Clean up numerous inconsistencies

### PR DESCRIPTION
**Pull request by jbcrail**
Summary of updates:
* Missing import from `bokeh/application/handlers/script.py`
* Align output of build artifacts in `setup.py`
* Match example document titles with example file name
* Add document titles to examples

### CODE REVIEW

File: bokeh/application/handlers/script.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from __future__ import absolute_import, print_function
 
 import os
+import sys
```
Comment (mattpap) [inline bokeh/application/handlers/script.py]: This is a bit worrying. Didn't we test imports? Or maybe we only test for unused imports?

**Comment by mattpap** [comment]
LGTM

**Comment by damianavila** [comment]
@bokeh/core, changes touch several files but in a benign way... I am towards to merge this one soon before tagging. Anyone against that?

**Comment by mattpap** [comment]
@damianavila, no, this PR doesn't have non-trivial changes.

**Comment by damianavila** [comment]
thanks @jbcrail !!

**Pull request merged**